components/fileselect/templates.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Templates
+page_title: FileSelect - Templates
+description: Templates in the FileSelect for Blazor.
```

Comment:
Aim at descriptions that are around 150 symbols. This is vital for the SEO of the page.
```suggestion
description: Discover the Blazor FileSelect component templates that enable you to customize the rendered button. The templates allow you to change the text and add custom content. 
```